### PR TITLE
ChemAxon.Marvin version 21.10

### PR DESCRIPTION
**Pull request by oxygen-dioxide**
- [ ] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [ ] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [ ] Have you validated your manifest locally with `winget validate --manifest <path>`? 
- [ ] Have you tested your manifest locally with `winget install --manifest <path>`?
- [ ] Does your manifest conform to the [1.0 schema](https://github.com/microsoft/winget-cli/blob/master/doc/ManifestSpecv1.0.md)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/17248)

### CODE REVIEW

File: manifests/c/ChemAxon/Marvin/21.10/ChemAxon.Marvin.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+- csrxn
+- cssdf
+- t
+- csrdf
+- mol
+Installers:
+- InstallerLocale: en-US
+  Architecture: x64
+  InstallerUrl: https://dl.chemaxon.com/marvin/21.10.0/marvin_windows-x64_21.10_openjdk11.exe
+  InstallerSha256: AE7340E2C24EEEDE40CD8F31B01B66D6FC2CB9B6C95FA5343C61D91AB619A0A3
```
Comment (ImJoakim) [inline manifests/c/ChemAxon/Marvin/21.10/ChemAxon.Marvin.installer.yaml:23]:
```suggestion
  InstallerSha256: 2C254E9FDD88AF0208FC33D7672BE99BE187C4A779DCC785E8FE1DCE0199A7FD
```

File: manifests/c/ChemAxon/Marvin/21.10/ChemAxon.Marvin.locale.en-US.yaml [hunk 2]
```diff
@@ -0,0 +1,19 @@
+# Created using wingetcreate
+
+PackageIdentifier: ChemAxon.Marvin
+PackageVersion: 21.10
```
Comment (ItzLevvie) [inline manifests/c/ChemAxon/Marvin/21.10/ChemAxon.Marvin.locale.en-US.yaml:4]:
```suggestion
PackageVersion: 21.10.0
```

File: manifests/c/ChemAxon/Marvin/21.10/ChemAxon.Marvin.installer.yaml [hunk 3]
```diff
@@ -0,0 +1,28 @@
+# Created using wingetcreate
+
+PackageIdentifier: ChemAxon.Marvin
+PackageVersion: 21.10
```
Comment (ItzLevvie) [inline manifests/c/ChemAxon/Marvin/21.10/ChemAxon.Marvin.installer.yaml:4]:
```suggestion
PackageVersion: 21.10.0
```

File: manifests/c/ChemAxon/Marvin/21.10/ChemAxon.Marvin.locale.en-US.yaml [hunk 4]
```diff
@@ -0,0 +1,19 @@
+# Created using wingetcreate
+
+PackageIdentifier: ChemAxon.Marvin
+PackageVersion: 21.10
+PackageLocale: en-US
+Publisher: ChemAxon
+Author: ChemAxon
+PackageName: Marvin
```
Comment (ItzLevvie) [inline manifests/c/ChemAxon/Marvin/21.10/ChemAxon.Marvin.locale.en-US.yaml:8]:
```suggestion
PackageName: ChemAxon Marvin Suite
```

File: manifests/c/ChemAxon/Marvin/21.10/ChemAxon.Marvin.yaml [hunk 5]
```diff
@@ -0,0 +1,7 @@
+# Created using wingetcreate
+
+PackageIdentifier: ChemAxon.Marvin
+PackageVersion: 21.10
```
Comment (ItzLevvie) [inline manifests/c/ChemAxon/Marvin/21.10/ChemAxon.Marvin.yaml:4]:
```suggestion
PackageVersion: 21.10.0
```

**Review by ItzLevvie — CHANGES_REQUESTED** [review]

**Comment by ImJoakim** [comment]
You need to update the version folder as well after changing version using GitHub Desktop.

**Review by ItzLevvie — APPROVED** [review]

**Pull request merged**